lib/cli/do_volapi.js

```diff
@@ -0,0 +1,393 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/*
+ * Copyright (c) 2016, Joyent, Inc.
+ */
+
+/*
+ * The 'sdcadm experimental volapi' CLI subcommand.
+ */
+
+var assert = require('assert-plus');
+var util = require('util'),
+    format = util.format;
+var vasync = require('vasync');
+
+var common = require('../common');
+var errors = require('../errors');
+var DownloadImages = require('../procedures/download-images').DownloadImages;
+var shared = require('../procedures/shared');
+
+var NB_MBS_IN_GB = 1024;
+var NFS_SHARED_VOLUMES_PACKAGES_NAME_PREFIX = 'nfs_shared_volume_';
```

Comment:
> FWIW, I'd prefer that names for things in the volumes world start with "vol" or "volume" instead of using "nfs".

Agreed.

> Nfs is mostly an implementation detail here, isn't it?

It depends on the perspective. Users will be able to create NFS shared volumes and mount them using the `mount` command, at which point they need to know the shared volume's storage is made available via an NFS interface: they'll need to use the `-t nfs` command line switch for the mount operation to work. 

We also chose the name `tritonnfs` for the Docker volume driver that corresponds to these volumes for the same reason. NFS is how the storage is made available through the network, and is thus part of the implementation, but it's also part of the interface.

If we agree on that, how would `volume_nfs` work as a package name prefix?